p2p/module.go

```diff
@@ -31,6 +32,49 @@ type p2pModule struct {
 	network typesP2P.Network
 }
 
+// HandleEvent implements modules.P2PModule
+func (m *p2pModule) HandleEvent(message *anypb.Any) error {
```

Comment:
I'm getting a bit worried that `p2p/module.go` is getting very big.

Do you think it would make it more readable to create a `p2p/event_handler.go` and `p2p/network_handler.go`?

Reply:
Resolving since its out of scope.